packages/cosmo/package.py

```diff
@@ -103,7 +103,10 @@ class Cosmo(MakefilePackage):
     conflicts('+production', when='~eccodes')
 
     conflicts('+cppdycore', when='%pgi cosmo_target=cpu')
-    build_directory = 'cosmo/ACC'
+    if 'COSMO_BUILD_DIR' in os.environ:
```

Comment:
why do we need this ? Until now environment is not part of the API, so I am not sure it is good idea to introduce parametrizations of the bevaiour via environment variables

Reply:
it is not clear how to do it without env